sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py

```diff
@@ -57,29 +57,48 @@ class CloudEvent(EventMixin):   #pylint:disable=too-many-instance-attributes
     :type source: str
     :param data: Event data specific to the event type.
     :type data: object
-    :param type: Required. Type of event related to the originating occurrence.
+    :keyword type: Required. Type of event related to the originating occurrence.
     :type type: str
```

Comment:
other kwargs type should also be annotated as Optional?
```suggestion
    :type type: Optional[str]
```

Reply:
type is required - CloudEvent has 3 required params - `source` , `type` and `data`

Reply:
updated it back to param - thanks for the catch :)